tests/containers/k3s_helm_install.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: k3s helm
+# Summary: Ensure that k3s and helm is installed in the system
+# - check if k3s is installed; if not, install k3s
+# - check if helm is installed; if not, install helm
+# Maintainer: qe-c <qe-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use utils;
+use Utils::Architectures;
+use serial_terminal qw(select_serial_terminal);
+use main_containers qw(is_suse_host);
+use containers::k8s qw(install_helm install_k3s);
+
+sub run {
+    select_serial_terminal;
+
+    die "Not supported architecture" unless is_aarch64() || is_x86_64();
+    die "Not a Suse host" unless is_suse_host();
+
+    install_k3s() if (script_run("which k3s") != 0);
```

Comment:
Let `install_k3s` do this [which](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/containers/k8s.pm#L85) for you:

```suggestion
    install_k3s();
```